### PR TITLE
user_reports added for sending mails  daily to users

### PR DESCRIPTION
**Pull request by jagadeesh6jaga**
# Description

# api endpoint : 

it will send  respective Reports through  mail to all annotators and reviewers   daily at  00:05 .

### CODE REVIEW

File: backend/user_reports.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+from projects.models import Project
+from datetime import datetime
+from users.views import AnalyticsViewSet
+from django.db.models import Q
+import pandas as pd
+from django.core.mail import send_mail
+from django.conf import settings
+
+
+def caluculate_reports():
```
Comment (GokulNC) [inline backend/user_reports.py:21]: Minor correction: there's a spelling mistake in "calculate"

**Review by GokulNC — APPROVED** [review]

File: backend/user_reports.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            [reviewer.email],
+        )
+
+
+# schedule.every(10).seconds.do(caluculate_reports)
+schedule.every().day.at("00:05").do(caluculate_reports)
+
+while 1:
+    schedule.run_pending()
+    time.sleep(1)
```
[the remark this answers is not in the document]
Comment (GokulNC) [reply backend/user_reports.py:131]: Yeah Ishvinder.. probably we can make the waiting time larger. Like 10mins or so.

File: backend/user_reports.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        df1 = pd.DataFrame.from_records(final_data["total_summary"], index=[0])
+        blankIndex = [""] * len(df1)
+        df1.index = blankIndex
+        email_to_send = {"ProjectWiseReport": df, "Total Reports Summary": df1}
+
+        # print(email_to_send)
+
+        send_mail(
+            "Your Annotation Reports",
+            f"Your Annotation Reports are:{email_to_send}",
```
Comment (ishvindersethi22) [inline backend/user_reports.py:84]: Use pretty-HTML-tables to beautify email.
Please refer to this: 
- https://dev.to/siddheshshankar/convert-a-dataframe-into-a-pretty-html-table-and-send-it-over-email-4663
- https://github.com/sbi-rviot/ph_table

**Review by ishvindersethi22 — APPROVED** [review]
Verified and Tested on dev server

**Pull request merged**